### PR TITLE
Add release 1.4.0 post

### PR DESCRIPTION
**Pull request by Scrumplex**
Needs to be updated with macOS signature once we actually release 1.4.0

### CODE REVIEW

File: src/download/windows.md [hunk 1]
```diff
@@ -24,10 +24,18 @@ eleventyNavigation:
 
 <div class="infobox top">
 
-## Advanced Windows install options
+## Legacy version
+
+There is a legacy version of PolyMC, primarily for older versions of Windows.
```
Comment (DioEgizio) [inline src/download/windows.md:29]:
```suggestion
There is a legacy version of PolyMC, primarily for older versions of Windows like Windows 7 or Windows 8.1.
```

File: src/news/release-1.4.0.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- Fix hangs when aborting some tasks by @flowln in #831
+- Fix missing instance version in status bar by @kthchew in #860
+- Fix permanent mod-loaders for some modpacks by @Scrumplex in #880
+- Fix "Play Offline" not working when auth servers are partially offline by @Scrumplex in #877
+- Fix sorting for Modrinth resources by @DioEgizio in #837
+- Fix Technic instances using Forge by @Scrumplex in #890
+- Fix translations by @Scrumplex in #917
+- Fix warnings reported by LGTM.com by @Scrumplex in #808
+- [Linux] Fix mods relying on `libstdc++` on Nix by @virchau13 in #904
+- [macOS] Workaround Qt bug to fix menu bar separators by @kthchew in #827
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:77]: Fun fact: that regression is fixed on qt 6.2 so people wouldn't even have noticed this

File: src/news/release-1.4.0.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+release_version: 1.4.0
+minimum_macos_version: 10.14.0
+mac_signature: epiclmao
+tags:
+  - release
+---
+
+Whoa! There have been a few things since our last feature release almost two months ago!
+
+Major new features include the new mod updater and Qt 6 support.
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:14]:
```suggestion
Major new features include the new mod updater and Qt 6 support.
If you still want to use Qt 5 don't worry, in the short term PolyMC will still support it and packages for it are available.
However, if you use Qt 6 your experience should be better overall, especially for HiDPI users.
```

Comment (Scrumplex) [reply src/news/release-1.4.0.md:14]: I mean we only say "Qt 6 support", not "switch to Qt 6", so I don't think anyone would misunderstand it.

Comment (DioEgizio) [reply src/news/release-1.4.0.md:14]: I think some people will think we're switching to Qt 6 in any case, better to clarify that.
And if they don't think we're switching to qt6, some will think we want to remove qt5 support asap, when we aren't

File: src/news/release-1.4.0.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- Fix translations by @Scrumplex in #917
+- Fix warnings reported by LGTM.com by @Scrumplex in #808
+- [Linux] Fix mods relying on `libstdc++` on Nix by @virchau13 in #904
+- [macOS] Workaround Qt bug to fix menu bar separators by @kthchew in #827
+
+### Removed
+ - Remove Offline Accounts in favour of "Play Offline" by @Scrumplex in #877
+
+
+**Full Changelog**: https://github.com/PolyMC/PolyMC/compare/1.3.1...1.3.2
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:86]: ah yes

Comment (DioEgizio) [reply src/news/release-1.4.0.md:86]:
```suggestion
**Full Changelog**: https://github.com/PolyMC/PolyMC/compare/1.3.2...1.4.0
```

File: src/news/release-1.4.0.md [hunk 5]
```diff
@@ -67,6 +68,7 @@ Thanks to all contributors, who have worked on this release!
 - Always scale mod icons to the right size by @flowln in #818
 - Don't censor offline access token by @ivanpu in #864
 - Don't create unnecessary folders when extracting ZIPs by @flowln in #817
+- Fix contrast of palceholder text with dark theme by @flowln in #929
```
Comment (flowln) [inline src/news/release-1.4.0.md:71]:
```suggestion
- Fix contrast of placeholder text with dark theme by @flowln in #929
```
palceholder xd

**Review by flowln — APPROVED** [review]

File: src/download/linux.md [hunk 6]
```diff
@@ -40,6 +40,15 @@ yay -S polymc-bin
 # latest git package:
 yay -S polymc-git
 ```
+
+If you want to use Qt 5 to build the packages instead:
+```
+# stable Qt 5 source package:
+yay -S polymc-qt5
+# latest Qt 5 git package:
+yay -S polymc-qt5-git
```
Comment (DioEgizio) [inline src/download/linux.md:49]: If https://github.com/PolyMC/PolyMC/pull/922 gets merged you'll have to add polymc-qt5-bin too btw

File: src/news/release-1.4.0.md (hunk 5 above)
Comment (Scrumplex) [reply src/news/release-1.4.0.md:71]: palceholder

File: src/news/release-1.4.0.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+## Changelog
+
+### Added
+- Add concurrent tasks by @flowln in #786
+- Add more links to mod and modpack descriptions by @flowln and @DioEgizio in #634 #799
+- Add news reader in the launcher by @flowln in #845
+- Add on-disk mod metadata information by @flowln and @Scrumplex in #426 #714 #716 #759
+- Implement mod updater by @flowln in #588
+- Preliminary work for modpack-updates by @jamierocks in #665
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:32]:
```suggestion
- Preliminary work for modpack updates by @jamierocks in #665
```

File: src/news/release-1.4.0.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+- Qt 6 support by @Scrumplex in #575 #902 #911 #936
+- [Linux] Add Performance settings (MangoHud, GameMode and PRIME) by @Jan200101 in #707
+- [Linux] Add more metadata to Nix package by @TheOPtimal in #930
+- [macOS] Implement updater using Sparkle by @kthchew and @Scrumplex in #479
+- [OpenBSD] Add support for OpenBSD by @Technous285 in #694
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:37]:
```suggestion
- [OpenBSD] Readd support for OpenBSD by @Technous285 in #694
```

File: src/news/release-1.4.0.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+### Changed
+- Allow the cape to be changed without changing the skin by @Gingeh in #913
+- Cleanup Java code by @Scrumplex and @icelimetea in #778 #785 #800 #863
+- Hide channel from version for tagged commits by @Scrumplex in #810
+- Improve handling of environment variables by @Scrumplex in #893
+- Improve resolving of JARs by @Scrumplex in #794
+- Improve the account selection window by @Gingeh and @Scrumplex in #855 #870
+- Improve UX in mod-downloader by @flowln in #505
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:46]:
```suggestion
- Improve UX in the mod downloader by @flowln in #505
```

File: src/news/release-1.4.0.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+- Improve UX in modpack-downloader by @flowln in #839
```
Comment (DioEgizio) [inline src/news/release-1.4.0.md:47]:
```suggestion
- Improvements for Modrinth/CurseForge modpacks by @flowln in #839
```

**Review by DioEgizio — COMMENTED** [review]
some nitpicks

**Pull request merged**